core/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
nitpick

If you have `build/` in your root `.gitignore` then you don't have to have a `.gitignore` for each module.

Related discussion: https://github.com/JuulLabs/juul-app-android/pull/1992#discussion_r406665454

_(screenshot for those without access):_

![Screen Shot 2020-07-09 at 3 04 15 PM](https://user-images.githubusercontent.com/98017/87095439-976ef100-c1f5-11ea-8b29-dc24d42d76d5.png)